product/tutorials/tutorial_set_inventory.md

```diff
@@ -0,0 +1,105 @@
+# **Set Inventory**
+
+## Let's get started
+
+Changes to a product's inventory information may occur much more frequently than changes to its catalog information.
```

Comment:
Changes to the product inventory information may occur more frequently than changes to the catalog information.